cypress/e2e/tests/custom-metrics/analysis_metrics.test.ts

```diff
@@ -28,88 +28,84 @@ const metricName = "konveyor_tasks_initiated_total";
 let applicationList: Array<Application> = [];
 let counter: number;
 
-describe(
-    ["@tier2"],
-    "Bug MTA-3320: Custom Metrics - Count the total number of initiated analyses",
-    function () {
-        before("Login", function () {
-            login();
-        });
+describe(["@tier2"], "Custom Metrics - Count the total number of initiated analyses", function () {
+    before("Login", function () {
+        login();
+    });
 
-        beforeEach("Load data and define interceptors", function () {
-            cy.fixture("application").then(function (appData) {
-                this.appData = appData;
-            });
-            cy.fixture("analysis").then(function (analysisData) {
-                this.analysisData = analysisData;
-            });
+    beforeEach("Load data and define interceptors", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
 
-            cy.intercept("GET", "/hub/application*").as("getApplication");
+        cy.intercept("GET", "/hub/application*").as("getApplication");
 
-            // Get the current counter value
-            metrics.getValue(metricName).then((counterValue) => {
-                counter = counterValue;
-            });
-            Application.open(true);
+        // Get the current counter value
+        metrics.getValue(metricName).then((counterValue) => {
+            counter = counterValue;
         });
+        Application.open(true);
+    });
 
-        it("Perform analyses - Validate the tasks initiated counter increased", function () {
-            // For source code analysis application must have source code URL git or svn
-            const bookServerApp = new Analysis(
-                getRandomApplicationData("bookserverApp", {
-                    sourceData: this.appData["bookserver-app"],
-                }),
-                getRandomAnalysisData(this.analysisData["source_analysis_on_bookserverapp"])
-            );
-            bookServerApp.create();
-            cy.wait("@getApplication");
-            cy.wait(2000);
-            bookServerApp.analyze();
-            bookServerApp.verifyAnalysisStatus("Completed");
-            counter++;
-            applicationList.push(bookServerApp);
+    it("Perform analyses - Validate the tasks initiated counter increased", function () {
+        // For source code analysis application must have source code URL git or svn
+        const bookServerApp = new Analysis(
+            getRandomApplicationData("bookserverApp", {
+                sourceData: this.appData["bookserver-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["source_analysis_on_bookserverapp"])
+        );
+        bookServerApp.create();
+        cy.wait("@getApplication");
+        cy.wait(2000);
```

Comment:
Doesn't the counter now increase on language and task discovery as well ?

Reply:
Yes, I need to modify the test. It will be done in another PR